obspy/core/inventory/inventory.py

```diff
@@ -332,35 +332,87 @@ def get_response(self, seed_id, datetime):
             raise Exception(msg)
         return responses[0]
 
-    def get_coordinates(self, seed_id, datetime=None):
+    def _get_channel_metadata(self, seed_id, datetime=None):
```

Comment:
Any particular reason this is a private method? Probably useful in its own right.